### DISCUSSION TITLE
Add multiblock to pyvista and pyvistaqt

### DISCUSSION THREAD

**Discussion by Gleb-GG** [pyvista/pyvistaqt#227] [Q&A] [answered]
Hi everyone and pyvista-team!

I have problems with add_mesh(multiblock). I wrote scripts with pyvista earlier and ,now, im creating gui with pyvistaqt. I set up multiblock of spheres and added in scene.

```python
spheres = pv.MultiBlock([pv.Sphere()]) 
p = pv.Plotter(notebook=False)
p.add_mesh(spheres)
```
It took 9.841379165649414 seconds.

In pyvistaqt
```python
self.plotter.add_mesh(self.Multiblock)
```
It took 59.311408042907715 seconds.

Why it happens? And how i can reduce time?

pv.Report()
--------------------------------------------------------------------------------
                OS : Windows
            CPU(s) : 4
           Machine : AMD64
      Architecture : 64bit
               RAM : 10.2 GiB
       Environment : Python
        GPU Vendor : Intel
      GPU Renderer : Intel(R) UHD Graphics 620
       GPU Version : 4.5.0 - Build 27.20.100.8681

           pyvista : 0.31.3
               vtk : 9.1.0
             numpy : 1.21.6
           imageio : 2.17.0
           appdirs : 1.4.4
            scooby : 0.5.12
            meshio : 4.4.6
        matplotlib : 3.4.3
         pyvistaqt : 0.9.0
             PyQt5 : 5.15.4
           IPython : 7.23.1
          colorcet : 3.0.0
             scipy : 1.8.0
              tqdm : 4.64.0
--------------------------------------------------------------------------------

**Comment by akaszynski** [comment, marked as answer]
This is being addressed in https://github.com/pyvista/pyvista/pull/2941 and will be out in `pyvista==0.37.0`. If you're feeling brave, feel free to checkout [feat/add_composite](https://github.com/pyvista/pyvista/tree/feat/add_composite).

**Reply by Gleb-GG** [reply]
My bad, I didn't say that I use QtInteractor. This class haven't metod 'add_composite' as I see.

**Reply by akaszynski** [reply]
It inherits from the pyvista.Plotter class, so when 0.37.0 comes out you'll have it.

You can also get it right now by installing from source and using the above branch.

**Reply by Gleb-GG** [reply]
Thanks!! It's amazing, It takes 3.1086599826812744 seconds.

**Reply by akaszynski** [reply]
Awesome. I'm trying to get this out in the release soon. Glad it works, please let me know if you encounter any issues at all as it's a significant refactor.
